css/product.css

```diff
@@ -1,5 +1,3 @@
-@import "/css/Utility.css";
-
 .list-heading label {
   margin: 0.5rem 0;
```

Comment:
you can use px if the value is less than 1rem

Reply:
Thanks for suggestion using rem Everywhere in project and comfortable with it